netmiko/base_connection.py

```diff
@@ -1140,7 +1140,7 @@ def select_delay_factor(self, delay_factor: float) -> float:
             else:
                 return self.global_delay_factor
 
-    def special_login_handler(self, delay_factor: float = 1.0) -> None:
+    def special_login_handler(self, delay_factor: float = 1.0) -> Optional[str]:
```

Comment:
I wouldn't expect this to change was there something that made you switch this? I would expect this to always return None.

Reply:
That was for `HuaweiSSH` class. `HuaweiSSH.special_login_handler` returns string.